ui/service.go

```diff
@@ -65,17 +67,27 @@ func New(things mainflux.ThingsServiceClient, sdk sdk.SDK) Service {
 	}
 }
 
+func parseTemplate(tmps ...string) (tpl *template.Template, err error) {
+	for _, tmp := range tmps {
+
+		tpl = template.New(tmp)
+		tpl = tpl.Funcs(template.FuncMap{
+			"toJSON": func(data map[string]interface{}) string {
+				ret, _ := json.Marshal(data)
+				return string(ret)
+			},
+		})
+		a := append(tmplFiles, templateDir+tmp)
```

Comment:
Use `tmlp`, because `tmp` is often shortened for `temporary`. Also - first append just `a := append(tmplFiles, tmpl)`, then in foreach add `templateDir` prefix o all array members.

What does `a` stands for, btw?